ha/core/event_manager/subscribe_event.py

```diff
@@ -16,11 +16,13 @@
 # cortx-questions@seagate.com.
 
 from typing import List
+from ha.k8s_setup import const
 from ha.core.event_manager.resources import RESOURCE_STATUS
-from ha.core.event_manager.resources import RESOURCE_TYPES
+from ha.core.event_manager.resources import RESOURCE_TYPES, FUNCTIONAL_TYPES
 
 class SubscribeEvent:
-    def __init__(self, resource_type : RESOURCE_TYPES, states : List[RESOURCE_STATUS]):
+    def __init__(self, resource_type : RESOURCE_TYPES, states : List[RESOURCE_STATUS],
+                 functional_types: list = []):
```

Comment:
Please change Args section in the function docstring.

Reply:
Done.